.travis.yml

```diff
@@ -7,10 +7,10 @@ php:
 matrix:
   include:
     -
-        php: 7
+        php: 7.1
```

Comment:
You set travis to 7.1 while composer still says 7.0. Make a choice :)

Reply:
This matrix only defines on which version of php it will run with the lowest / highest dependencies. It will run still run on 7.0 and 7.1 with the dependencies from the lock file

Reply:
Had some more issues with this, as the lock file assumes php 7.1.

It now runs everything on 7.0 with the lowest possible dependencies and on 7.1 with both the lowest and highest.